packages/yoastseo/src/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -137,18 +154,24 @@ PreviouslyUsedKeyword.prototype.scoreAssessment = function( previouslyUsedKeywor
  * @returns {{id: number, count: number}} The object with the count and the id of the previously used keyword
  */
 PreviouslyUsedKeyword.prototype.researchPreviouslyUsedKeywords = function( paper ) {
-	var keyword = paper.getKeyword();
-	var count = 0;
-	var id = 0;
+	const keyword = paper.getKeyword();
+	let count = 0;
+	let postTypeToDisplay = "";
+	let id = 0;
 
 	if ( ! isUndefined( this.usedKeywords[ keyword ] ) ) {
 		count = this.usedKeywords[ keyword ].length;
+
+		if ( count > 0 ) {
+			postTypeToDisplay = this.usedKeywordsPostTypes[ keyword ][ 0 ];
+		}
 		id = this.usedKeywords[ keyword ][ 0 ];
```

Comment:
maybe I'm missing something but shouldn't this line also be inside the if statement above?